src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -207,7 +209,7 @@ public void removeNation() {
 		if (!hasNation())
 			return;
 		
-		Nation nation = this.nation;
+		Nation nulling = this.nation;
```

Comment:
Please change this back to nation.

Reply:
Will do. FYI: this was intentional, as it was masking the field.

Reply:
Yeah I figured it was, but it makes things less readable to me, as it is no longer apparent what it represents.

Reply:
Used oldNation instead. That should cover both the readability, and the non-masking bases.